packages/textlint/package.json

```diff
@@ -94,10 +94,10 @@
     "textlint-rule-preset-jtf-style": "^2.3.11",
     "ts-node": "^9.1.1",
     "ts-node-test-register": "^9.0.1",
-    "typescript": "~4.0.2"
+    "typescript": "~4.2.4"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Comment:
It aims to support ESM env
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c